### PR TITLE
Fix storybook bug for Sitewide Links

### LINKED ISSUE #2451: Sitewide Links component links no longer change in Storybook
**Describe the bug**
In Storybook, the Sitewide Links component has a list of links, with default being in English and saying "Link x". However, changing the service only updates the text around it and not the links, which are sometimes necessary to see if the component is going to break with longer text or any other WS specific criteria.

Currently: 
image

Previously:
image

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to https://bbc.github.io/psammead/?path=/story/components-sitewidelinks--default
2. Select a service

**Expected behaviour**
Links change as before.

**Additional context**
Most likely caused by changes made [here](https://github.com/bbc/psammead/pull/2404/commits/1e71e267a229fc1e0e7bb7c0604780940988ab6e#diff-9a6daa50f38a2ed6254f1e71f42a1a43L12-L23)

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #2451 

**Overall change:** 
Previously in Storybook for the Sitewide Links component names of links were not updating for different services and stayed as "Link x". This was an issue since it is useful to see if the component is going to break with longer text or any other WS specific criteria. Now the text in links changes for different services.

Previously:
Screenshot 2019-10-22 at 12 20 18

Currently:
Screenshot 2019-10-22 at 12 23 39

**Code changes:**
- Checked the current service before filling in an array with link's text.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-sitewide-links/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.1.17 | [PR#2457](https://github.com/bbc/psammead/pull/2457) Fix storybook bug with links for services |
```
Comment (ghost) [inline packages/components/psammead-sitewide-links/CHANGELOG.md]:
```suggestion
| 3.1.17 | [PR#2457](https://github.com/bbc/psammead/pull/2457) Show relevant link text when selecting service in storybook |
```
I think this explains what has been done in this PR.

**Review by ghost — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by AlistairGempf — APPROVED** [review]
Looks good

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**